sdk/core/core-amqp/src/cbs.ts

```diff
@@ -144,7 +144,7 @@ export class CbsClient {
     } catch (err) {
       err = translate(err);
       logger.warning(
-        "[%s] An error occured while establishing the cbs links: %O",
```

Comment:
So here's a funny situation. If a customer sends us logs and we go and search for the message in our code we won't be able to find it anymore.

Might not be a big deal as long as we're aware that misspellings have been corrected :)

CC: @ramya-rao-a , @chradek , @HarshaNalluru

Reply:
I hear you, but if anything, I'd be worried about this one: https://github.com/Azure/azure-sdk-for-js/pull/8956/files#diff-025fc300e6be927698de02bdaf01856cR66

Reply:
If this was for `@azure/amqp-common`, then I would have the same concerns. This is because the number of existing customers using Track 1 EH & SB in production would be much larger than the ones for Track 2 at this point in time. Right now, this concerns customers who have EH Track 2 in production, and am ok to bite the bullet and get it done.